agent/version/version.go

```diff
@@ -22,10 +22,10 @@ package version
 // repository. Only the 'Version' const should change in checked-in source code
 
 // Version is the version of agent
-const Version = "1.88.0"
+const Version = "1.89.0"
 
 // GitDirty indicates the cleanliness of the git repo when this agent was built
 const GitDirty = true
 
 // GitShortHash is the short hash of this agent build
-const GitShortHash = "c9040caf"
+const GitShortHash = "9494fb35"
```

Comment:
This seems to point to commit `9494fb35`, however the latest commit in dev branch is `4843cd00` which I believe is the one that we want. 

I suspect this happened due to release PR being generated under a stale local branch which has not yet pulled the `4843cd00` commit changes from upstream. If I'm not mistaken, I think you just need to pull in the latest changes from dev into your local branch and re-perform the internal process to regenerate the `version.go` file(s).

Reply:
Got it. Taking a look

Reply:
The branch wasn't stale but I forgot to rerun the internal process and regenerate the version files. Version files are regenerated now, pointing to `4843cd00`